odoo_backup_sh/models/odoo_backup_sh.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Stanislav Krotov <https://it-projects.info/team/ufaks>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+import json
+import werkzeug
+
+import odoo
+from odoo import http, api, fields, models
+from odoo.http import request
+from odoo.addons.odoo_backup_sh.controllers.main import BackupController, BACKUP_SERVICE_ENDPOINT
+from odoo.addons.iap import jsonrpc, InsufficientCreditError
+
+
+class Dashboard(models.Model):
```

Comment:
Are you sure that it cannot be ``models.TransientModel`` or even ``models.AbstractModel``?